cpp/example_code/iot/describe_endpoint.cpp

```diff
@@ -0,0 +1,84 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Before running this C++ code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html
+ *
+ * For information on the structure of the code examples and how to build and run the examples, see
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html.
+ *
+ **/
+
+#include <aws/core/Aws.h>
+#include <aws/iot/IoTClient.h>
+#include <aws/iot/model/DescribeEndpointRequest.h>
+#include <iostream>
+#include "iot_samples.h"
+
+// snippet-start:[cpp.example_code.iot.DescribeEndpoint]
+//! Describe the endpoint specific to the AWS account making the call.
```

Comment:
IS ! char required?

Reply:
This is a symbol that is used to generate documents with doxygen. It means "brief description". It is not used currently.